Ska/engarchive/fetch.py

```diff
@@ -69,6 +70,31 @@
 # Cached version (by content type) of first and last available times in archive
 CONTENT_TIME_RANGES = {}
 
+# Data source. Default to CXC. Set to MAUDE by fetch.from_maude()
+_backend = 'CXC'
+
+def from_cxc():
+    maude.handler.disconnect()
```

Comment:
This can go in the backend class in the `set()` method.  The main thing I don't fully understand with this session management is whether there are consequences to `disconnect()` without a `connect()` or multiple `connect()` in a row, or connecting and then doing nothing for 2 days in an interactive Python shell.  @gmrehbein - do you know?